src/main/java/org/folio/sidecar/integration/keycloak/KeycloakClient.java

```diff
@@ -11,20 +11,24 @@
 import io.vertx.ext.web.client.HttpResponse;
 import io.vertx.ext.web.client.WebClient;
 import jakarta.enterprise.context.ApplicationScoped;
-import lombok.RequiredArgsConstructor;
+import jakarta.inject.Named;
 import lombok.extern.log4j.Log4j2;
 import org.folio.sidecar.integration.keycloak.configuration.KeycloakProperties;
 import org.folio.sidecar.model.ClientCredentials;
 import org.folio.sidecar.model.UserCredentials;
 
 @Log4j2
 @ApplicationScoped
-@RequiredArgsConstructor
 public class KeycloakClient {
 
   private final WebClient webClient;
   private final KeycloakProperties properties;
 
+  public KeycloakClient(@Named("webClient") WebClient webClient, KeycloakProperties properties) {
```

Comment:
you can add `@Named` annotation to the `lombok.config` and continue using `@RequiredArgsConstructor`